card/mod/core/lib/tasks/card.rake

```diff
@@ -91,17 +97,27 @@ namespace :card do
   end
 
   def add_opt letter, key, desc
-    op.on "-#{letter}", "--#{key.to_s.tr '_', '-'} #{key.to_s.upcase}", desc do |val|
+    op.on "-#{letter}", key_to_option_description(key), desc do |val|
       options[key] = val
     end
   end
 
+  def add_opt_without_shortcut key, desc
+    op.on key_to_option_description(key), desc do |val|
+      options[key] = val
+    end
+  end
+
+  def key_to_option_description key
+    "--#{key.to_s.tr '_', '-'} #{key.to_s.upcase}"
+  end
+
   def op
     @op ||= OptionParser.new
   end
 
   def parse_options task
-    op.banner = "Usage: rake card:#{task} -- [options]"
+    op.banner = "Usage: card #{task} [options]"
```

Comment:
In theory this should show "decko" when you run `decko eat` and "card" when you run `card eat`. But this is clearly a step in the right direction.